proj/src/main/webapp/index.html

```diff
@@ -11,19 +11,32 @@
         <!-- TODO: Add SDKs for Firebase products that you want to use
             https://firebase.google.com/docs/web/setup#available-libraries -->
         <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-analytics.js"></script>
+
+        <script src="app.js"></script>
```

Comment:
Is the problem of when to load this script fixed?

Reply:
Yes, based on lee's comment, the initialization scripts are in the header while the logic scripts are at the bottom of body